From 5802227309d036664580ba114b609aee1e97d425 Mon Sep 17 00:00:00 2001
From: Rock Lee <rli@sierrawireless.com>
Date: Wed, 10 Jan 2018 21:08:24 -0500
Subject: [PATCH 1/4] ubifs: remove error message in ubifs_xattr_get

commit b3e73839379327ee21240ac48ea9a2eaf7613d79 upstream

There is a situation that other modules, like overlayfs, try to get
xattr value with a small buffer, if they get -ERANGE, they will try
again with the proper buffer size. No need to report an error.

Signed-off-by: Rock Lee <rli@sierrawireless.com>
Signed-off-by: Richard Weinberger <richard@nod.at>
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 fs/ubifs/xattr.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/fs/ubifs/xattr.c b/fs/ubifs/xattr.c
index 98f1125..901cc69 100644
--- a/fs/ubifs/xattr.c
+++ b/fs/ubifs/xattr.c
@@ -385,8 +385,6 @@ ssize_t ubifs_xattr_get(struct inode *host, const char *name, void *buf,
 	if (buf) {
 		/* If @buf is %NULL we are supposed to return the length */
 		if (ui->data_len > size) {
-			ubifs_err(c, "buffer size %zd, xattr len %d",
-				  size, ui->data_len);
 			err = -ERANGE;
 			goto out_iput;
 		}
-- 
1.7.9.5

